guides/v2.2/cloud/project/project-conf-files_services-mysql.md

```diff
@@ -55,40 +94,19 @@ mysqldb:
             importer:
                 privileges:
                     main: rw
-
 ```
 
-In the preceding example, the endpoint (that is, user) `reporter` has `ro` privileges to the `main` database and endpoint `importer` has `rw` access to the `main` database. This means that:
+In the preceding example, the endpoint (user) `reporter` has read-only privileges to the `main` database and endpoint `importer` has read-write access to the `main` database. This means that:
 
 -  The `admin` user has full control of the database.
 -  The `repoter` user has SELECT privileges only.
 -  The `importer` user has SELECT, INSERT, UPDATE, and DELETE privileges.
 
-## Add MySQL in services.yaml and .magento.app.yaml {#settings}
-
-To enable MySQL, add the following code with your installed version and allocated disk space in MB to `.magento/services.yaml`.
-
-```yaml
-mysql:
-    type: mysql:10.0
-    disk: 2048
-```
-
-To configure the relationships for the environment variable, set a relationship in your `.magento.app.yaml` in the Git branch. For example:
+The endpoints defined in the above example must also exist in the `relationships` property of the `.magento.app.yaml` file. For example:
```

Comment:
Not sure:  If you add them here, you have to update the .magento.app.yaml file? --  If that's the case, consider -- If you add additional endpoints, you must also add them to the `relationships` section in the .magento.app.yaml... 

Or verify that the relationships are also included in the ...